recipes/main/schnitzel_viennese_style.adoc

```diff
@@ -0,0 +1,48 @@
+[id='sec.schnitzel_viennese_style']
+
+
+ifdef::env-github[]
+:imagesdir: ../../images
+endif::[]
+ifndef::env-github[]
+:imagesdir: images
+endif::[]
+
+=== Schnitzel Viennese Style
+
+image::schnitzel_viennese_style/schnitzel.jpg[Name, 640, 480]
+
+Contributed by: Robert
+image:contributors/robert_s.png[Name, 140, 140]
+
+Schnitzel is very common in Germany, the original however is from Austria. Schnitzel Viennese (Wiener Schnitzel) has to be made of veal, the more popular made of pork (rarely turkey) is called Viennese Style (Wiener Art). Making is very much the same and since it's crumbed, there's usually no sauce with it.
+
+Ingredients (4 servings):
+
+[width="100%",cols="3,3,10,10",options="header"]
+|=========================================================
+|Metric | US    | Ingredient         | Zutat
+
+| 4     |       | pork escalopes     | Schweineschnitzel
+|       |       | starch flour       | Stärkemehl 
+| 2     |       | eggs               | Eier 
+|       |       | breadcrumbs        | Semmelbrösel
+|       |       | salt               | Salz
+|       |       | pepper             | Pfeffer 
+|       |       | resolidified butter| Butterschmalz
+| 1     |       | lemon              | Zitrone
+|=========================================================
+
+Preparation:
+
+. Dry the schnitzels with a paper towel, put them on a wooden cutting board and treat them with a meat tenderiser, as thin as you like them. Or as big as your pan is.
+. Prepare three plates or bowls, first with with starch flour, for the second mix the eggs and add a good amount of salt and pepper, the third gets the breadcrumbs.
+. Pull the schnitzels first through the starch so they are covered and the egg will better stick.
+. Pull the schnitzels through the eggs, they must be fully covered, no dry spots.
+. Pull the schnitzels through the breadcrumbs and press on, cover them completely. 
```

Comment:
```suggestion
. Put the schnitzels into the last bowl and press onto them, covering them completely in breadcrumbs. 
```

Reply:
I actually pull them through, not put them in :-)